src/commands/paint.ts

```diff
@@ -97,15 +97,19 @@ export function paint(
     process.stdout.write(`${chalk.bold('Snowpack')}\n\n`);
     // Dashboard
     if (devMode) {
-      process.stdout.write(
-        `  ${chalk.bold.cyan(`${devMode.protocol}//localhost:${devMode.port}`)}`,
-      );
-      for (const ip of devMode.ips) {
-        process.stdout.write(
-          `${chalk.cyan(` > `)}${chalk.bold.cyan(`${devMode.protocol}//${ip}:${devMode.port}`)}`,
-        );
+      // SERVER_START event happened
+      if (startTimeMs > 0 && port > 0 && protocol) {
+        process.stdout.write(`  ${chalk.bold.cyan(`${protocol}//localhost:${port}`)}`);
+        for (const ip of ips) {
+          process.stdout.write(
+            `${chalk.cyan(` > `)}${chalk.bold.cyan(`${protocol}//${ip}:${port}`)}`,
+          );
+        }
+        process.stdout.write('\n' + chalk.dim(`  Server started in ${startTimeMs}ms.\n\n`));
+      } else {
+        // still waiting for SERVER_START
+        process.stdout.write(chalk.dim(`  Server starting…`) + '\n\n');
```

Comment:
When the server is starting (line break added):

![Screen Shot 2020-06-12 at 12 45 12 PM](https://user-images.githubusercontent.com/1369770/84536218-986e3a80-acaa-11ea-9155-47db8d5f28e6.png)

Reply:
Just to confirm, if there's no reinstall needed, this is only shown for ~20ms right?

Reply:
Right. But there are other events happening, so in some instances that’s helpful because it’ll be visible for longer. But if there’s nothing to be done yeah it’s so quick you might not even notice that flash.

Reply:
tbh I kind of like that behavior, just like "Server startin--- SERVER STARTED FASTER THAN YOU COULD EVEN READ THOSE TWO WORDS"